tests/cli/test_orion.py

```diff
@@ -56,3 +56,38 @@ def test_start_with_keep_alive_from_setting(mock_run_process: AsyncMock):
     command: List[str] = mock_run_process.call_args[1]["command"]
     assert "--timeout-keep-alive" in command
     assert command[command.index("--timeout-keep-alive") + 1] == "100"
+
+
+def test_start_from_cli_with_keep_alive(mock_run_process: AsyncMock):
+    invoke_and_assert(
+        ["orion", "start", "--keep-alive-timeout", "100"],
+        expected_output_contains=[
+            "prefect config set PREFECT_API_URL=http://127.0.0.1:4200/api",
+            "View the API reference documentation at http://127.0.0.1:4200/docs",
+        ],
+    )
+
+    mock_run_process.assert_awaited_once()
+    command: List[str] = mock_run_process.call_args[1]["command"]
+    assert "--timeout-keep-alive" in command
+    assert command[command.index("--timeout-keep-alive") + 1] == "100"
+
+
+def test_start_from_cli_with_invalid_keep_alive(mock_run_process: AsyncMock):
+    invoke_and_assert(
+        ["orion", "start", "--timeout-keep-alive", "45"],
+        expected_code=2,
+        expected_output_contains=[
+            "No such option: --timeout-keep-alive Did you mean --keep-alive-timeout?",
+        ],
+    )
```

Comment:
We could probably skip this test entirely — this kind of thing is covered by the `typer` tests. Maybe write a test about what happens if an invalid value is passed instead? Like "-10" or "foo"?

Reply:
@madkinsz It looks like prefect Setting allows to specify a negative value and passes it along to uvicorn and uvicorn does not complain and starts. What is unclear is that what would be the effective value in this case.

Tested this with uvicorn directly and it lets a negative value through and starts normally. I did take a quick look at uvicorn repo but did not find any checks being made for a non negative value. I have not looked at the code for the transport layer to see if it has any check or what the behavior is with a value of < 1.

Because the Setting type is declared to be int, it does complain when using a non-numeric value. 

I can add an exit_with_error with a check for value > 5 in src/prefect/cli/orion.py around L99 ?

Reply:
Thanks for investigating! I don't think we need to validate a positive integer if Uvicorn doesn't — let's see if anyone has problem first.

Reply:
I have removed this test